azurerm/data_source_express_route_circuit_test.go

```diff
@@ -0,0 +1,47 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func testAccDataSourceAzureRMExpressRoute_basicMetered(t *testing.T) {
```

Comment:
since this is private (as the methods starts with a lower-case character) - I've pushed a commit to add this to the resource test, I hope you don't mind

Reply:
thanks for doing this!